src/lib.rs

```diff
@@ -464,25 +484,18 @@ fn pan_orbit_camera(
                 pan_orbit.target_focus += translation;
                 has_moved = true;
             }
-        } else if scroll.abs() > 0.0 {
-            let apply_scroll = |value: f32| value - scroll * value * 0.2;
-
-            if let Projection::Orthographic(ref mut p) = *projection {
-                p.scale = util::apply_zoom_limits(
-                    apply_scroll(p.scale),
-                    pan_orbit.zoom_upper_limit,
-                    pan_orbit.zoom_lower_limit,
-                );
-            } else {
-                pan_orbit.radius = pan_orbit.radius.map(|radius| {
-                    util::apply_zoom_limits(
-                        apply_scroll(radius),
-                        pan_orbit.zoom_upper_limit,
-                        pan_orbit.zoom_lower_limit,
-                    )
-                });
-            }
-            has_moved = true;
+        } else if (scroll_line + scroll_pixel).abs() > 0.0 {
+            // Calculate the amount of change based on line scroll and pixel scroll
+            let line_delta = -scroll_line * pan_orbit.target_radius * 0.2;
+            let pixel_delta = -scroll_pixel * pan_orbit.target_radius * 0.2;
+            // Add the calculated deltas to the target radius
+            pan_orbit.target_radius = util::apply_zoom_limits(
```

Comment:
You are only applying zoom limits to the line scrolling, and not the pixel scrolling

Reply:
> You are only applying zoom limits to the line scrolling, and not the pixel scrolling

Yes, I have applied `zoom limits` to `pan_orbit.target_radius + line_delta + pixel_delta`, which includes `pixel scrolling`.